examples/express-composition/src/migrate.ts

```diff
@@ -0,0 +1,20 @@
+import {NoteApplication} from './application';
+
+export async function migrate(args: string[]) {
+  const existingSchema = args.includes('--rebuild') ? 'drop' : 'alter';
+  console.log('Migrating schemas (%s existing schema)', existingSchema);
+
+  const app = new NoteApplication();
```

Comment:
Should we use `const app = new ExpressApplication()` here instead? Since it's a wrapper/facade of our Note application IIUC.